controller/linodemachine_controller.go

```diff
@@ -675,16 +693,10 @@ func (r *LinodeMachineReconciler) reconcileDelete(
 		return nil
 	}
 
-	if machineScope.LinodeCluster.Spec.Network.LoadBalancerType != "dns" {
-		if err := services.DeleteNodeFromNB(ctx, logger, machineScope); err != nil {
-			logger.Error(err, "Failed to remove node from Node Balancer backend")
-			return err
-		}
-	} else {
-		if err := services.DeleteIPFromDNS(ctx, machineScope); err != nil {
-			logger.Error(err, "Failed to remove IP from DNS")
-			return err
-		}
+	if err := r.removeMachineFromLB(ctx, logger, machineScope); err != nil {
+		logger.Error(err, "Failed to remove machine to LB")
```

Comment:
```suggestion
		logger.Error(err, "Failed to remove machine from LB")
```

Reply:
Done... Also updated another place with the same grammatical error